website/docs/BestPractices.md

```diff
@@ -214,3 +214,76 @@ If abstracted away it could look something like below where the logic is put in
 // 👍
 await form.submitData(new Person('bob@webdriver.io'))
 ```
+
+## File Downloads
```

Comment:
Can we move this into an own subpage as this topic is big enough to cover a whole page.

Reply:
Create a new directory called `best-practices` and put `FileDownloads.md` in there. Then you replace [`"bestpractices",`](https://github.com/webdriverio/webdriverio/blob/828c7578fc55dae54362525d7b40f514f0a4ff87/website/_sidebars.json#L45) with:
```json
        {
          "type": "category",
          "label": "Best Practices",
          "link": {
            "type": "doc",
            "id": "bestpractices"
          },
          "items": [
            "best-practices/file-download"
          ]
        },
```

Note that the `id` in `FileDownloads.md` should be `id: file-download`.

Reply:
@christian-bromann What should i do with BestPractice.md file that is already over there?

Reply:
ok. one moment. will this open the Best Practice.md when i click on best practice but it will also give me a dropdown for file downloads section?

Reply:
yes

Reply:
That's awesome, bro!